packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -278,6 +278,11 @@ fi
 %endif
 
 %changelog
+* Thu Mar 19 2020 Sharanya Devaraj <sharanyd@amazon.com> - 1.38.0-1
```

Comment:
line 34-35 of this file needs to be updated

Reply:
done